README.md

```diff
@@ -41,7 +41,9 @@ Running your app using IPFS will mimic the production environment that will be u
 - **build:app**: Builds the front-end
 - **build:script**: Builds the background script
 - **test**: Runs tests for the contracts
-- **publish**: Builds the apps and the contracts and publishes them to IPFS and APM
+- **publish:content**: Builds the apps publishes to IPFS
+- **publish:minor**: Builds the apps and the contracts and publishes them to IPFS and APM
+- **publish:major**: Builds the apps and the contracts and publishes them to IPFS and APM. Deployed contract will default to the contract name specified in the arapp.json file
```

Comment:
I'd reword these comments:

- `publish:content`: Release a minor version to aragonPM with just the new content
- `publish:minor`: Release a minor version to aragonPM
  - ⚠️ **warning**: since we only allow contract upgrades on major versions, isn't this the same as `publish:content`?
- `publish:major`: Release a major version to aragonPM with a potentially new contract address for on-chain upgrades.
  - ⚠️**warning**: if the contract doesn't change, does aragonCLI automatically reuse the same contract address or will it always redeploy the contract?

Reply:
You are right `publish:minor` and `publish:content` are doing the same. About major, right now, aragonCLI is creating new address every time even if contract its the same